examples/rosa-classic-with-unmanaged-oidc/main.tf

```
@@ -89,15 +90,15 @@ module "rosa_cluster_classic" {
 # machine-pool
 ############################
 module "rosa_machine_pool" {
-  source = "../../modules/machin-pool"
+  source = "../../modules/machine-pool"
 
-  cluster_id = module.rosa_cluster_classic.cluster_id
-  name = var.machine_pool_name
+  cluster_id   = module.rosa_cluster_classic.cluster_id
+  name         = "${var.cluster_name}-MachinePool"
```

please use lower case 
```suggestion
  name         = "${var.cluster_name}-machine-pool"
```

---

Fixed